### PR TITLE
Update Panel.java

### PR DESCRIPTION
**Pull request by juliachin123**
Changed how the command from the command line is parsed and passed to the command interface

### CODE REVIEW

**Review by evelez7 — COMMENTED** [review]
LGTM

**Review by netcables — COMMENTED** [review]
LGTM

File: src/codeu/chat/client/commandline/Panel.java [hunk 1]
```diff
@@ -48,10 +48,10 @@ public void register(String commandName, Command command) {
   // will be returned. True will be return if a command is found. Whether or not
   // the command was successful is not returned.
   //
-  public boolean handleCommand(String commandName, Scanner line) {
+  public boolean handleCommand(String commandName, List<String> args) {
```
Comment (Tuxdude) [inline src/codeu/chat/client/commandline/Panel.java:51]: I think you will need to import `java.util.List` for this code to compile. Also does `Command.invoke()` now accept `List<String>` ?

**Review by Tuxdude — COMMENTED** [review]
I'm guessing you're making small changes per commit which is actually a good practice. +1 to that. 

Another equally important thing to keep in mind is that each commit should compile and not break the build. Imagine what might happen if one PR gets merged but the other one does not, or if someone picks up only one commit and not the other since it was not submitted yet.

If the changes are related and cannot be broken down into smaller commits without breaking the build, I'd recommend combining them. It also helps with code reviews since the reviewers can understand the context better regarding why a certain line was changed.

**Pull request closed without merging**